METIS/METIS_DET_IFU.yaml

```diff
@@ -9,6 +9,9 @@ properties:
     temperature: -233
     dit: "!OBS.dit"
     ndit: "!OBS.ndit"
+    mindit: 0.04              # seconds, E-TNT-MPIA-1004, v1-0
```

Comment:
For the LMS detectors, the fast readout mode is currently not baselined. Therefore, min_dit = 1.3 seconds (even though these are physically the same detectors as the one in the IMG-LM, which we run at up to 25 Hz full frame. Reason: fast readout is more noisy and gives much higher power (thermal dissipation) at the detector).

Reply:
Changed to 1.3 s (already in branch `dev_master`)